src/sentry/event_manager.py

```diff
@@ -2109,8 +2109,8 @@ def _handle_regression(group: Group, event: BaseEvent, release: Release | None)
 
     # we only mark it as a regression if the event's release is newer than
     # the release which we originally marked this as resolved
-    elif GroupResolution.has_resolution(group, release):
-        return None
+    # elif GroupResolution.has_resolution(group, release):
+    #     return None
```

Comment:
I'm forcing the issue to happen.

In my test case, we end up reaching this comparison:
https://github.com/getsentry/sentry/blob/159d52c5273b0d6b326e3b2669635cd4a880d0a7/src/sentry/models/groupresolution.py#L156-L172